src/bank.rs

```diff
@@ -97,17 +100,13 @@ pub struct Bank {
     // The latest confirmation time for the network
     confirmation_time: AtomicUsize,
 
-    // Mapping of account ids to Subscriber ids and sinks to notify on userdata update
-    account_subscriptions: RwLock<HashMap<Pubkey, HashMap<Pubkey, Sink<Account>>>>,
-
-    // Mapping of signatures to Subscriber ids and sinks to notify on confirmation
-    signature_subscriptions: RwLock<HashMap<Signature, HashMap<Pubkey, Sink<RpcSignatureStatus>>>>,
-
     /// Tracks and updates the leader schedule based on the votes and account stakes
     /// processed by the bank
     pub leader_scheduler: Arc<RwLock<LeaderScheduler>>,
 
     pub storage_state: StorageState,
+
+    subscriptions: RwLock<Arc<RpcSubsciptions>>,
```

Comment:
Can this be a BankSubscriptions instead of RpcSubscriptions, to avoid the bank depending on rpc_pubsub.rs